crates/biome_js_analyze/src/lint/complexity/no_useless_fragments.rs

```diff
@@ -190,7 +194,17 @@ impl Rule for NoUselessFragments {
                                 }
                             }
                             JsSyntaxKind::JSX_TEXT => {
-                                if !child.syntax().text().to_string().trim().is_empty() {
+                                let child_text =
+                                    child.syntax().text().to_string().trim().to_string();
```

Comment:
There's no need to allocate two strings. I think `text().trim()` should work

Reply:
done